Percy/Percy.cs

```diff
@@ -245,38 +262,48 @@ public static void Screenshot(WebDriver driver, string name, IEnumerable<KeyValu
                             userOptions["consider_region_elements"] = elementIds;
                         }
                     }
+                    userOptions[SYNC] = isSync;
```

Comment:
isn't this will set value of sync false even when true is passed by user?

Reply:
removed